File: codewars/elixir/return_negative.exs
defmodule Negator do
  def make_negative(num) do
    if num < 0 do
      num
    else
      num * - 1
    end
  end
end

IO.puts(Negator.make_negative(2))
IO.puts(Negator.make_negative(12))
IO.puts(Negator.make_negative(22))
IO.puts(Negator.make_negative(0))
IO.puts(Negator.make_negative(-2))
IO.puts(Negator.make_negative(-12))
IO.puts(Negator.make_negative(-22))
IO.puts(Negator.make_negative(-1))

File: codewars/c/return_negative.c
// https://www.codewars.com/kata/55685cd7ad70877c23000102/train/c

#include <stdio.h>

int makeNegative(int num) {
    if (num < 0) {
        return num;
    }

    return num * -1;
}

int main(int argc, char *argv[]) {
    printf("%d\n", makeNegative(2));
    printf("%d\n", makeNegative(12));
    printf("%d\n", makeNegative(22));
    printf("%d\n", makeNegative(0));
    printf("%d\n", makeNegative(-2));
    printf("%d\n", makeNegative(-12));
    printf("%d\n", makeNegative(-22));

    return 0;
}
